lore/species/tajaran/The Great Frost.md

```diff
@@ -0,0 +1,85 @@
+
+Modern Day
+
+With the aid of outside groups supplying the holds with rare materials they could not produce, the creation of more hearth engines, the success of the arboretums, the frost breakers, and the now well armed and hardened populace united under S�randarr life became worth living. Much of the focus on many tajarans has been brought under the preservation of old traditions and expanding into the world with the making of new hearth engines. Priests guided by Sun Wardens traveled to old temples, towns, and cities searching for ancient scrolls of S�randarr. Artwork, writings, and any artifact of significance was taken and brought back to the hearth engines to once again be preserved in the light of S�randar�s grace. The bodies found were given funeral rites and the dead properly dealt with out of respect. With all eyes turned towards these noble goals civil strife was rare, unity prevailed.
+
+Nerahni
+The Nerahni once again took up their tradition of art and creation. The great frost had killed many of them and the survivors found purpose in expression. Though much of their art became grim and macabre their was a hope in what they made that effected all who saw it. The design of structures, temples, frost breakers, and the paintings they made decorate much of the holds. Those among them who did not become artist founds themselves pushed into practical positions with the creation of architecture and stone masonry before they slowly reclaimed their former lofty position.
+
+Selem
+The Selem struggled greatly with their identity even as order was reestablished. Decades of failures and the dying of many great men left them collectively as shells of their former selves. The rebuilding of themselves and their culture was a slow and treacherous political game now that only the cunning and treacherous remained in power as the cult of personalities that once held them together had crumbled. Only in recent years have good men of worth begun to once more appear among them, behind these men other Selem fell into place. Slowly but surely the greatness they once held is coming back.
+
+Siik
+The Siik and their struggle was not of the external end but an internal one. Many Siik carry the feeling within them that S�randarr and Messa care nothing for them and that life and happiness is only within them alone. Few Siik will publicly reject the gods but many when praying praise the collective strength of the people and not S�randarr. A larger number among them has also chosen to leave the planet and find themselves among other worlds, a not unheard of trait for a young Siik tajaran to have never set foot on Adhomai.

```

Comment:
"... publicly reject the gods, but many, when praying, praise the..."
"... larger number among them have also chosen..."
"... find themselves among other worlds. It's not unheard of for a young Siik..."